mmv1/templates/terraform/pre_create/compute_instance_settings_fingerprint.go.tmpl

```diff
@@ -0,0 +1,21 @@
+
+fingerprintUrl, err := tpgresource.ReplaceVars(d, config, "{{"{{"}}ComputeBasePath{{"}}"}}projects/{{"{{"}}project{{"}}"}}/zones/{{"{{"}}zone{{"}}"}}/instanceSettings/{{"{{"}}name{{"}}"}}")
```

Comment:
```suggestion
url, err := tpgresource.ReplaceVars(d, config, "{{"{{"}}ComputeBasePath{{"}}"}}projects/{{"{{"}}project{{"}}"}}/zones/{{"{{"}}zone{{"}}"}}/instanceSettings/{{"{{"}}name{{"}}"}}")
```

Reply:
This would overwrite the normal POST/PATCH URLs prepared above this code, that will be used below this code